### PR TITLE
[WIP] - allow inquiry_questionnaire modals to be closed via x button, click outside or escape

### PR DESCRIPTION
**Pull request by oxaudo**
This code change adds 'x' button to all inquiry dialogs. And also removes 'nevermind' links from all of those dialogs since this link is now redundant.

screen shot 2018-04-16 at 10 32 35 am

This ended up being substantially more code kill than I anticipated. It's a bit tricky now to track who actually worked on all of those files originally since the history only includes move to `src` directory. Would appreciate if maybe @xtina-starr or maybe @damassi can take a look here and flag any potential problems they could see with all this code removals.

### CODE REVIEW

**Comment by alloy** [comment]
@oxaudo This PR is pointing to the wrong branch, it should go into `feature-add-accepts-card-payments-indicator`

screen shot 2018-04-13 at 16 37 26

**Comment by alloy** [comment]
(I deleted my previous comment, because I wasn’t paying close attention. This PR was probably meant to go into `master` rather than the QA deployment.)

**Comment by alloy** [comment]
The changes look good to me thus far. Once you hear back and no further changes are needed, feel free to merge 👍

**Comment by oxaudo** [comment]
Yea, this one is unrelated to credit card display so should be treated independently. I confirmed that we are ok with removing ‘never mind’ link and all associated events and will add that removal here a bit later. I know that Eloy will be out next two weeks so will ask somebody else to take a look before merging. All of that will probably happen on Monday:)

File: src/desktop/analytics/inquiry_questionnaire.js [hunk 1]
```diff
@@ -1,210 +1,212 @@
...
-  $document.on('click', '.js-choice', function () {
+  $document.on('click', '.js-choice', function() {
     var choice = $(this).data('value')
     track('Clicked on how_can_we_help option', {
-      choice: choice
+      choice: choice,
     })
   })
 
-  $document.on('click', '.js-nevermind', function (e) {
```
Comment (oxaudo) [inline src/desktop/analytics/inquiry_questionnaire.js]: This is the actual change. The rest is just reformatting that happened on save.

File: src/desktop/analytics/inquiry_questionnaire.js [hunk 2]
```diff
@@ -1,210 +1,212 @@
...
+    )
   })
 
-  bindOnce('contact:submitted', function (context) {
+  bindOnce('contact:submitted', function(context) {
     analytics.track('Contact form submitted', context.attributes)
   })
 
-  bind('inquiry:sent', function (context) {
+  bind('inquiry:sent', function(context) {
```
Comment (damassi) [inline src/desktop/analytics/inquiry_questionnaire.js]: (Assuming all of the above is just Prettier changes)

**Comment by sweir27** [comment]
👍 looks good to me!

**Pull request merged**